linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitor.java

```diff
@@ -133,7 +136,7 @@ public static void main(String[] arguments)
       } catch (ModelBuildingException ex) {
         // When there's a pom.xml file that is not (indirectly) referenced by the root pom.xml,
```

Comment:
I'm not sure what "not (indirectly)" means. Is this the same as "not" or a double negative? Can you clarify this at all?

Reply:
Updated the comment.